### PR TITLE
remove all first quest related roles upon adding member to server

### PR DESCRIPTION
**Pull request by tikisailor**
I hope this fixes the issue that users who re-enter the DAO server while already having FQ roles, receive scrambled FQ messages. Please note that I was not able to test this yet as I could not get through the verification getting triggered on multiple bots. It is fairly simple though and if it is ok with you to merge it into develop I can give it a quick test on Mechanical Animal before release. Thanks @amaredeus for finding this bug.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 1. Remove datadog tracer
 2. Add more phrases and fix grammar and add typing feature
+3. First Quest: bug fix to prevent message scramble in egde case
```
Comment (amaredeus) [inline CHANGELOG.md:7]: egde -> edge

**Review by amaredeus — CHANGES_REQUESTED** [review]

**Review by nonsensecodes — APPROVED** [review]
Thank you, @tikisailor!

**Review by nonsensecodes — CHANGES_REQUESTED** [review]
One more thing: the version number in the `package.json` as requested by @amaredeus

**Comment by nonsensecodes** [comment]
Requested branch consolidation as more work is incoming.

**Pull request closed without merging**